Makefile

```diff
@@ -160,3 +161,7 @@ post-release: tools/yq
 .PHONY: generate
 generate: tools/stringer
 	go generate ./...
+
```

Comment:
```suggestion
	.bin/ory dev headers license --exclude=.bin --exclude=internal/httpclient --exclude=proto

```

As now the generated files also have headers, we need to run the licencing command after generate.

Reply:
I think a better course of action is calling `make format` here. This way the formatting logic remains all in one place.